host_core/lib/application.ex

```diff
@@ -101,7 +101,8 @@ defmodule HostCore.Application do
       {HostCore.Actors.ActorSupervisor,
        strategy: :one_for_one,
        allow_latest: config.allow_latest,
-       allowed_insecure: config.allowed_insecure},
+       allowed_insecure: config.allowed_insecure,
+       enable_actorfromfs: config.enable_actorfromfs},
```

Comment:
nit: I think `enable_actor_from_fs` would be a good name here to keep everything consistent